cmd/variables.go

```diff
@@ -72,7 +78,7 @@ func (h *Handler) VariablesSet(ctx context.Context, req *entity.CommandRequest)
 		updatedEnvNames = append(updatedEnvNames, key)
 	}
 
-	err = h.ctrl.UpsertEnvs(ctx, variables, &serviceName)
+	serviceID, err := h.ctrl.UpsertEnvs(ctx, variables, &serviceName)
```

Comment:
Would rather have an extra call to get the serviceId from serviceName than to change the function signature here

Reply:
I have been thinking about something similar but went naively with that approach since `UpsertEnvs` does a lot of steps to obtain the serviceID which is why I wanted to reuse the value it just computed. Can you elaborate what alternative you had in mind?

Reply:
Hoist the work required to get serviceId from name into another function, call that function above, then pass in the serviceId if you want

Reply:
I see. The same snippet to obtain the serviceID by name was actually copy & pasted multiple times across the codebase already. I'll have a look at it, but can't promise anything since it's also used in unrelated parts of the codebase.

Reply:
Oh you don't have to refactor all of it. I'd just refactor it inline here and then we can move to pulling the usage into one function at some point

Reply:
Alright, I came with something, but wasn't fully sure where to put it exactly to not copy & paste even more. Hope that this is better.

Reply:
Looks perfect great job!